app/src/components/AddBusiness.js

```diff
@@ -0,0 +1,122 @@
+import React, { Fragment, useState, useRef } from 'react';
+
+
+
+
+const AddContact = () => {
+
+    const [name, setName] = useState("");
+    const [city, setCity] = useState("");
+    const [country, setCountry] = useState("");
+    const [lift, setLift] = useState("");
+
+    const [formValidity, setFormValidity] = useState({});
+
+    const onSubmitListing = async (e) => {
+        e.preventDefault();
+
+        const isEmpty = (currentValue) => currentValue === "";
+            
+            //console.log(body.every(isEmpty));
+            const body = { name, city, country, lift };
+                const updatedValidity = {
+
+                }
+
+            Object.entries(body).forEach(([key, value]) => {
+                if(isEmpty(value)){
+                    updatedValidity[key] = "invalid";
+                }
+
+
+            })
+            console.log(updatedValidity);
+
+
+        if(Object.keys(formValidity).length !== 0) {
+            return;
+        }
+
+        try {
+
+            
+            
+            console.log({ firstname, lastname })
+            const response = await fetch("http://localhost:3000/businesses", {
```

Comment:
Wondering if this is working? If you're following George's set up, I think you can use 
` const response = await fetch("/api/businesses", { ` instead. If you're not using his set up, can ignore!